src/__tests__/generate.test.ts

```diff
@@ -79,61 +79,102 @@ describe("The `generate` command", () => {
     expect(componentIndexExists).toBe(true);
     expect(componentTsxExists).toBe(true);
   });
+  describe("react-native-typescript", () => {
+    it("component template works and uses the default src/components path flag", async () => {
+      const componentName = "Reactnativetypescriptcomponent";
```

Comment:
Should we change this since now it will use Pascal case?

```suggestion
      const componentName = "ReactNativeTypeScriptComponent";
```

Reply:
@jsjoeio Technically that is still correct, it doesn't know to change those others to uppercase, I tested that out. So I can if you want, I wasn't sure.

Reply:
@cmejet Yes, you're right, both are correct. But if you _hadn't_ made your change, then "ReactNativeTypeScriptComponent" would result in -> "Reactnativetypescriptcomponent" and the test would fail. 

So my thinking, is to use Pascal case in the test, because this is what we expect.

Reply:
Sounds good!